ports/stm32/boards/stm32f091_af.csv

```diff
@@ -1,90 +1,90 @@
-PortE,PE12,TIM1_CH3N         ,SPI1_NSS/I2S1_WS    ,                    ,          ,                    ,                ,        ,         ,,,,,,,,,
-PortE,PE13,TIM1_CH3          ,SPI1_SCK/I2S1_CK    ,                    ,          ,                    ,                ,        ,         ,,,,,,,,,
-PortE,PE14,TIM1_CH4          ,SPI1_MISO/I2S1_MCK  ,                    ,          ,                    ,                ,        ,         ,,,,,,,,,
-PortE,PE15,TIM1_BKIN         ,SPI1_MOSI/I2S1_SD   ,                    ,          ,                    ,                ,        ,         ,,,,,,,,,
-PortF,PF0 ,CRS_SYNC          ,I2C1_SDA            ,                    ,          ,                    ,                ,        ,         ,,,,,,,,,
-PortF,PF1 ,                  ,I2C1_SCL            ,                    ,          ,                    ,                ,        ,         ,,,,,,,,,
-PortF,PF2 ,EVENTOUT          ,USART7_TX           ,USART7_CK/USART7_RTS,          ,                    ,                ,        ,         ,,,,,,,,,
-PortF,PF3 ,EVENTOUT          ,USART7_RX           ,USART6_CK/USART6_RTS,          ,                    ,                ,        ,         ,,,,,,,,,
-PortF,PF6 ,                  ,                    ,                    ,          ,                    ,                ,        ,         ,,,,,,,,,
-PortF,PF9 ,TIM15_CH1         ,USART6_TX           ,                    ,          ,                    ,                ,        ,         ,,,,,,,,,
-PortF,PF10,TIM15_CH2         ,USART6_RX           ,                    ,          ,                    ,                ,        ,         ,,,,,,,,,
-PortF,PF11,                  ,                    ,                    ,          ,                    ,                ,        ,         ,,,,,,,,,
+Port ,Pin ,AF0                                                                                 ,AF1                                                                  ,AF2                                   ,AF3                    ,AF4                           ,AF5                                  ,AF6     ,AF7      ,AF8 ,AF9 ,AF10,AF11,AF12,AF13,AF14,AF15,ADC      
+     ,    ,CAN1/CEC/CRS/EVENTOUT/I2S1/2/IR/RCC/SPI1/2/SYS/TIM1/3/14/15/16/17/TSC/USART1/2/3/4/8,CEC/EVENTOUT/I2C1/2/I2S1/2/IR/SPI1/2/TIM3/15/TSC/USART1/2/3/4/5/6/7/8,EVENTOUT/TIM1/2/16/17/USART5/6/7/8/USB,EVENTOUT/I2C1/TIM15/TSC,CAN1/CRS/I2C1/TIM14/USART3/4/5,I2C2/I2S2/RCC/SPI2/TIM15/16/17/USART6,EVENTOUT,COMP1/2  ,    ,    ,    ,    ,    ,    ,    ,    ,ADC      
+PortA,PA0 ,                                                                                    ,USART2_CTS                                                           ,TIM2_CH1/TIM2_ETR                     ,TSC_G1_IO1             ,USART4_TX                     ,                                     ,        ,COMP1_OUT,    ,    ,    ,    ,    ,    ,    ,    ,ADC1_IN0 
```

Comment:
The current files don't pad the last column. (Many people would have their editors configured to remove this, so this is going to lead to accidental diffs).